apps/sertra/components/footer.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, PropsWithChildren } from "react";
+
+export const Footer: FC<PropsWithChildren> = () => {
+  return (
+    <footer style={{ backgroundColor: "#17213B" }}>
+      <div style={{ width: "1140px", marginLeft: "auto", marginRight: "auto" }}>
```

Comment:
Is it a deliberate choice to use regular CSS instead of Chakra, and if so, is it intended to be for the MVP or a permanent choice?

Reply:
Was the fastest way to get it done, but not meant to be a permanent choice. At the very least inline styles should be replaced with some better way to style components